src/launcher/standalone_launcher/nginx/nginx.conf

```diff
@@ -0,0 +1,63 @@
+# nginx.conf  --  docker-openresty
```

Comment:
Delete this file. We have this in client. We'll lift nginx and all the Docker configuration out later.